corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -505,6 +506,8 @@ def get_report_element(
     row_index = 0
     rows = report_data_cache.get_data(report_config.uuid, data_source)
     for row_index, row in enumerate(rows):
+        if row_index + 1 >= settings.MAX_MOBILE_UCR_SIZE:
+            raise MobileUCRTooLargeException
```

Comment:
I assumed that `rows` was a generator rather than a list that already held all rows in memory, but that assumption might be incorrect.

Reply:
Fixed in 6838092eb4c399b641d0c7b1fec2cacefcdff6f5